pom.xml

```diff
@@ -293,9 +204,6 @@
 							<goal>integration-test</goal>
 						</goals>
 						<configuration>
-							<includes>
```

Comment:
Please make sure your project can be imported to WDT and Run as -> Run on Server.